.github/scripts/oss.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
As these aren't scripts is there a better name for the directory?

Reply:
Perhaps `matrices` and we could have the files be `enos-run-oss.json` and `enos-run-ent.json`?

Reply:
how about `enos-run-matrices` directory and `oss.json` and `ent.json` files ?